Project.toml

```diff
@@ -6,6 +6,7 @@ version = "0.5.12-pre"
 [deps]
 CodeTracking = "da1fd8a2-8d9e-5ec2-8556-3022fb5608a2"
 ConstructionBase = "187b0558-2788-49d3-abe0-74a17ed4e7c9"
+DiffEqCallbacks = "459566f4-90b8-5000-8ac3-15dfb0a30def"
```

Comment:
Is this a light or a heavy dependency? When I last checked, it depended on OrdinaryDiffEq.jl...

Reply:
It does not depend on OrdinaryDiffEq:
https://github.com/SciML/DiffEqCallbacks.jl/blob/master/Project.toml